RELEASING.md

```diff
@@ -21,8 +21,50 @@ the release script to actually perform the release and prepare main for the next
 where `<YOUR_KEY_SHORT_ID>` is the short id of your PGP key
 and `<RELEASE_VERSION>` is a release number such as `0.6.0`.
 
-Once the release is completed, use the [Nexus UI](https://s01.oss.sonatype.org/) to close the staging repository, then release it. That will push the Maven artefacts available
-on Maven Central.  The artefacts will take a while to reach all the Maven Central mirrors.
+The script will push the Maven artefacts to a Nexus **staging repository** and it will open a PR that will commit the release to Git.
+
+The next steps are:
+1.  Validate the staged artefacts.
+2.  If the staging artefacts are **good**, release the stage repository.  If they are not, **drop** the stage repository and start the release over.
+3.  Merge the PR.
+
+These are described below.
+
+### Validating the staged artefacts
+
+You can validate the staged artefacts by having a test application (we'll call it `T`) that uses kroxylicious-junit5-extension use the Maven artefacts by making
```

Comment:
Should we just have a stub project in GitHub that we use for validating the release?

I had loosely hoped at one stage to use the Sample plug-in for the task but I think I had missed the close step being required for publicly accessible staging repo URL

Reply:
Stub projects (one that consumes the junit ext and one that consumes the filter api) would make sense I think, but I haven't given it too much thought.